src/components/Card.js

```diff
@@ -0,0 +1,27 @@
+import Image from "next/image";
+
+export const Card = ({ card, onClick}) => {
+    return (
+        <div className={`perspective-1000 ${!card?.images ? "cursor-pointer" : 'pointer-events-none'}`} onClick={onClick}>
+            <div className={`w-[226px] h-[314px] relative transition-transform origin-center ${card?.image ? 'rotate-y-180' : ''}`} style={{ transformStyle: "preserve-3d" }}>
```

Comment:
Why do we use inline style here?

Reply:
Due to the fact that I used tailwind for the first time, and was in a hurry, I only figured out the plugins later, I will fix it